docs/source/en/api/pipelines/stable_diffusion/controlnet.mdx

```diff
@@ -0,0 +1,103 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Text-to-Image Generation with ControlNet Conditioning
+
+## StableDiffusionControlNetPipeline
+
+ControlNet by [@lllyasviel](https://huggingface.co/lllyasviel) is a neural network structure to control diffusion models by adding extra conditions.
+
+There are 8 pre-trained ControlNet models that were trained to condition the original Stable Diffusion model on different inputs, 
+such as edge detection, scribbles, depth maps, semantic segmentations and more.
+
+Using the pretrained models we can provide control images (for example, a depth map) to control Stable Diffusion text-to-image generation so that it follows the structure of the depth image and fills in the details.
+
+The original codebase/paper can be found here: 
+- [Code](https://github.com/lllyasviel/ControlNet)
+- [Paper](https://arxiv.org/abs/2302.05543)
```

Comment:
Please refer to [this documentation](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/pix2pix) as a reference to see how we usually structure the pipeline documentation. 

In particular, "Overview" should be the first sub-section. Under that, we include the paper name, the abstract, and other resources.

Reply:
Thanks! Already applied in this update. https://github.com/huggingface/diffusers/pull/2407/commits/06bb1db313d1e3b87d82d662ae02245a6894f30f